app/models/task/Task.scala

```diff
@@ -467,10 +467,9 @@ object TaskDAO {
       tasks <- Fox.combined(tasksSQL.map(Task.fromTaskSQL(_)))
     } yield tasks
 
-  def findAllByProject(projectName: String)(implicit ctx: DBAccessContext) =
+  def findAllByProject(projectId: ObjectId)(implicit ctx: DBAccessContext) =
```

Comment:
I would prefer this to have a Project as an parameter instead of the id. But maybe this is a pattern, as we only have the id in some cases, then never mind.

Reply:
If extracted from the URL, we only have the ID (in some cases we perform an object lookup anyway, just to check existance, but not always, because of the performance overhead). 
Generally, I would like to move towards using just the IDs in more cases, to make everything more lightweight. The queries *need* only the IDs, by their nature